requirements/test.in

```diff
@@ -3,7 +3,7 @@
 
 coverage
 mypy; implementation_name == "cpython"
-proxy.py
+proxy.py==2.4.4rc4
```

Comment:
Instead of pinning, use ` >= 2.4.4rc4`. With such an install request, pip (and by extension, pip-tools) automatically allows pre-releases for this package.
I think, originally, I was using something like `pip-compile -P 'proxy.py ~= 2.4.3rc0'` — the compatible release operator caps version. But I don't think that's necessary anymore. `>=` should be more than enough.

OTOH, maybe it'd work even without a change in this file, if you run `pip-compile` with `-P`, but I haven't checked. Dependabot would probably upgrade that to stable eventually, so we likely do need it here.

Reply:
adjusted in https://github.com/aio-libs/aiohttp/pull/7927/commits/be40981ac0096a3dbc6131db421e72d7b355fd51